scrapers.js

```diff
@@ -1686,6 +1686,36 @@ let scrapers = [
       };
     }
   },
+  {
+    country: 'ESP',
+    url: 'https://opendata.arcgis.com/datasets/48fac2d7de0f43f9af938852e3748845_0.csv',
+    priority: 1,
+    scraper: async function() {
+      let data = await fetch.csv(this.url);
+      let states = [];
+      for (let row of data) {
+        let state = row.Texto !== undefined ? parse.string(row.Texto) : "";
+        let cases = row.TotalConfirmados !== undefined ? parse.number(row.TotalConfirmados) : 0;
+        let deaths = row.TotalFallecidos !== undefined ? parse.number(row.TotalFallecidos) : 0;
+        let recovered = row.TotalRecuperados !== undefined ? parse.number(row.TotalRecuperados) : 0;
+        if(state !== "") {
```

Comment:
So what are rows with no state? Are they unassigned cases?

Reply:
They are always there in the data, I was mostly making sure the column "Texto" exists.